gepetuto/generate.py

```diff
@@ -87,4 +99,5 @@ def generate_ipynb(ipynb, folder, force_load=False):  # noqa: C901
                 elif dest is not None:
                     content.append(line)
     with ipynb.open("w") as f:
-        f.write(json.dumps(data, indent=1))
+        json.dump(data, f)
```

Comment:
```suggestion
        json.dump(data, f, indent=1)
```
It really helps with reviews :)